apps/condo/domains/meter/schema/MeterReading.test.js

```diff
@@ -309,12 +336,17 @@ describe('MeterReading', () => {
                 accountNumber: accountNumber1,
                 unitName,
             })
-            await updateTestProperty(adminClient, property.id, { deletedAt: 'true' })
-            const [source] = await MeterReadingSource.getAll(adminClient, { id: CALL_METER_READING_SOURCE_ID })
+            await PropertyAPI.softDelete(adminClient, property.id)
 
-            await expectToThrowAccessDeniedErrorToObj(async () => {
+            // NOTE: give worker some time
+            await sleep(1000)
+
+            const [source] = await MeterReadingSource.getAll(adminClient, { id: CALL_METER_READING_SOURCE_ID })
+            const testFunc = async () => {
```

Comment:
could you please don't change these cases! please create new tests for residents.
It's an access test! you don't need to wait a worker for access tests

Reply:
fixed